### PR TITLE
EK0 kronecker and diagonal solvers

### PR DESCRIPTION
**Pull request by pnkraemer**

### CODE REVIEW

File: tornadox/solvers/ek0_diagonal.py [hunk 1]
```diff
@@ -0,0 +1,192 @@
...
+        u0,
+        rtol,
+        atol,
+        safety=0.95,
+        factor_min=0.2,
+        factor_max=10.0,
+        power_integral_unscaled=0.3,
+        power_proportional_unscaled=0.4,
+    ):
+        del df  # unused
```
Comment (nathanaelbosch) [inline tornadox/solvers/ek0_diagonal.py:51]: remove? ^^

File: tornadox/solvers/ek0_kronecker.py [hunk 2]
```diff
@@ -0,0 +1,188 @@
...
+        u0,
+        rtol,
+        atol,
+        safety=0.95,
+        factor_min=0.2,
+        factor_max=10.0,
+        power_integral_unscaled=0.3,
+        power_proportional_unscaled=0.4,
+    ):
+        del df  # unused
```
Comment (nathanaelbosch) [inline tornadox/solvers/ek0_kronecker.py:52]: ?

File: tornadox/solvers/ek0_kronecker.py [hunk 3]
```diff
@@ -0,0 +1,188 @@
...
+        rtol,
+        atol,
+        safety=0.95,
+        factor_min=0.2,
+        factor_max=10.0,
+        power_integral_unscaled=0.3,
+        power_proportional_unscaled=0.4,
+    ):
+        """Perform a successful step."""
+        del df  # unused
```
Comment (nathanaelbosch) [inline tornadox/solvers/ek0_kronecker.py:89]: ?

File: tornadox/solvers/ek0_diagonal.py (hunk 1 above)
Comment (pnkraemer) [reply tornadox/solvers/ek0_diagonal.py:51]:
```suggestion
```

File: tornadox/solvers/ek0_diagonal.py [hunk 4]
```diff
@@ -0,0 +1,192 @@
...
+        rtol,
+        atol,
+        safety=0.95,
+        factor_min=0.2,
+        factor_max=10.0,
+        power_integral_unscaled=0.3,
+        power_proportional_unscaled=0.4,
+    ):
+        """Perform a successful step."""
+        del df  # unused
```
[the remark this answers is not in the document]
Comment (pnkraemer) [reply tornadox/solvers/ek0_diagonal.py:92]:
```suggestion
```

File: tornadox/solvers/ek0_kronecker.py (hunk 2 above)
Comment (pnkraemer) [reply tornadox/solvers/ek0_kronecker.py:52]:
```suggestion
```

File: tornadox/solvers/ek0_kronecker.py (hunk 3 above)
Comment (pnkraemer) [reply tornadox/solvers/ek0_kronecker.py:89]:
```suggestion
```

**Review by nathanaelbosch — APPROVED** [review]

**Pull request merged**